lib/src/cupertino/cupertino_popover_menu.dart

```diff
@@ -54,6 +56,18 @@ class CupertinoPopoverMenu extends SingleChildRenderObjectWidget {
   /// Color of the menu background.
   final Color backgroundColor;
 
+  /// The z-coordinate relative to the parent at which to place this popover.
```

Comment:
Let's change this comment to:

```dart
/// The virtual distance between this menu and the content that sits beneath it, which determines
/// the size, opacity, and spread of the menu's shadow.
///
/// The value must be non-negative.
```

Let's update similar dart docs elsewhere. If the term "menu" or other parts of this doc don't fit in the other situations, then let's update that wording when you replicate this dart doc, elsewhere.